hamt/hamt.go

```diff
@@ -81,13 +77,16 @@ type Shard struct {
 
 // NewShard creates a new, empty HAMT shard with the given size.
 func NewShard(dserv ipld.DAGService, size int) (*Shard, error) {
+	return NewShardWithHashFunc(dserv, size, mh.MURMUR3_128)
```

Comment:
Let's use 0x22 here for now. This variable's name is currently misleading in the global package and that will be resolved independently.